test/USCC.t.sol

```diff
@@ -0,0 +1,262 @@
+import "test/USTB.t.sol";
+import { USCC } from "src/USCC.sol";
+import { USCCV2 } from "test/USCCV2.sol";
+import { AllowList } from "src/AllowList.sol";
+
+contract USCCTest is USTBTest {
```

Comment:
This test file is almost exactly the same as the USTB one. I just overrode the tests that I needed to change the permission structs for

Reply:
if / when we do more tokens we prob want a way to run to DRY and the same tests for new tokens without a new file

Reply:
Yeah, we can cross that bridge when we come to it